modules/reference/pages/rpk/rpk-transform/rpk-transform-build.adoc

```diff
@@ -0,0 +1,43 @@
+= rpk transform build
+:description: rpk transform build
+
+Build a transform.

+

+This command looks in the current working directory for a transform.yaml file.

+

+Then depending on the language, will install the appropriate build plugin then 

+build a .wasm file.

+

+Extra arguments are passed directly to the underlying toolchain when it invoked.

+For example to add debug symbols and use the asyncify scheduler for tinygo:

+

+  rpk transform build -- -scheduler=asyncify -no-debug=false


```

Comment:
@rockwotj @micheleRP  this whole section is hard to understand. What are my options wrt TinyGo and the scheduler and goroutines?   What are tinygo 'release builds'? What does using the asyncify schedule enable me to do as a user?

Let's recall that are average user has no clue what TinyGo IS.    

We need to be very clear about option A vs B vs C and what you get with each one and cut out all irrelevant information wrt the runtime environment & language constructs the build option will support

Reply:
Let's move some of this discussion to https://github.com/redpanda-data/redpanda/pull/15055